config/content.d/support.rackspace.com.json

```diff
@@ -1,14 +1,14 @@
 {
     "support.rackspace.com": {
         "content": {
-            "/": "https://github.com/rackerlabs/docs-support-network/",
-            "/launch/": "https://github.com/rackerlabs/docs-support-launch/",
-            "/how-to/": "https://github.com/rackerlabs/rackspace-how-to/",
+            "/": "https://github.com/rackerlabs/docs-support-network/prod/",
+            "/launch/": "https://github.com/rackerlabs/docs-support-launch/prod/",
+            "/how-to/": "https://github.com/rackerlabs/rackspace-how-to/prod/",
             "/search/": null
         },
         "proxy": {
-            "/favicon.ico": "https://c0e9a73362c5d06f19d6-b39ff74cb91d713ac1253c5ae00eaccb.ssl.cf1.rackcdn.com/UI/favicon/favicon.ico",
-            "/images/favicon.ico": "https://c0e9a73362c5d06f19d6-b39ff74cb91d713ac1253c5ae00eaccb.ssl.cf1.rackcdn.com/UI/favicon/favicon.ico"
+            "/favicon.ico": "https://8d8dcdd952aa2708c2ff-519cda130c91226e76017ae910bdb276.ssl.cf1.rackcdn.com/favicon-aa186bee158ecea4c9b6a98e2ceec3141b6b7d8d94dcb71731ba112e2b17b1db.ico",
```

Comment:
Edward, just want to confirm - are these favicon changes expected?

Reply:
Updated to reflect master version